Libraries/LibWeb/Layout/InlineLevelIterator.cpp

```diff
@@ -293,7 +293,8 @@ Optional<InlineLevelIterator::Item> InlineLevelIterator::next_without_lookahead(
             x = tab_stop_dist.to_float();
         }
 
-        auto glyph_run = Gfx::shape_text({ x, 0 }, letter_spacing.to_float(), chunk.view, chunk.font, text_type);
+        auto shape_features = create_and_merge_font_features();
```

Comment:
This method doesn't have an implementation until the next commit. Please keep things in order so that everything builds and runs after each commit.